src/test/java/com/training/services/tests/VoyagesTest.java

```diff
@@ -89,20 +94,22 @@ public void testVoyageShipCodeValidate() {
     @Test
     public void testVoyageMasterSailDateValidate() {
         Response voyageResponse = getResponse();
-        logger.info("status--->" + voyageResponse.getStatus());
+
         List<Voyages> voyages = voyageResponse.getPayload().getVoyages();
-        logger.info("totalVoyagesCount-->" + voyages.size());
+
         Assertions.assertThat(voyageResponse.getStatus()).isEqualTo("200")
                 .as(" status is not 200");
-        for (int index = 0; index < voyages.size(); index++) {
+
+        IntStream.range(0, voyages.size()).forEach(index -> {
             String sailDate = voyages.get(index).getSailDate();
-            logger.info("sailDate->" + sailDate + " " + "index valuesS1-->" + index);
+
             softAssert.assertThat(sailDate.equals(voyages.get(index).getMasterSailDate()
                     .getMaster1SailDate())).isTrue()
                     .as("sailDate and master1SailDate of voyages" + index + "are not equal");
+
             softAssert.assertThat(sailDate.equals(voyages.get(index).getMasterSailDate()
                     .getMaster2SailDate())).isTrue()
                     .as("sailDate and master2SailDate of voyages" + index + "are not equal");
-        }
+        });
```

Comment:
Do we really need index in this?
Why cant we use - response.getPayload().getVoyages().forEach()?